client/src/components/TestSuite/TestSuiteDetails/TestListItem/TestListItem.tsx

```diff
@@ -102,58 +137,48 @@ const TestListItem: FC<TestListItemProps> = ({
 
   return (
     <>
-      <Box className={styles.listItem}>
-        <ListItem>
-          {test.result && (
-            <div className={styles.testIcon}>
-              <ResultIcon result={test.result} />
-            </div>
-          )}
-          <ListItemText primary={testLabel} />
-          {messagesBadge}
-          {requestsBadge}
-          {view === 'run' && runTests && (
-            <TestRunButton
-              runnable={test}
-              runnableType={RunnableType.Test}
-              runTests={runTests}
-              testRunInProgress={testRunInProgress}
-            />
-          )}
-          {expandButton}
-        </ListItem>
-        {test.result?.result_message && (
-          <ReactMarkdown className={styles.resultMessageMarkdown}>
-            {test.result.result_message}
-          </ReactMarkdown>
-        )}
-      </Box>
-      {view === 'run' && (
-        <Collapse in={open} className={styles.collapsible} unmountOnExit>
-          <Divider />
+      <Accordion
+        disableGutters
+        className={styles.accordion}
+        expanded={open}
+        TransitionProps={{ unmountOnExit: true }}
+        onClick={() => setOpen(!open)}
+      >
+        <AccordionSummary
+          aria-controls={`${test.title}-header`}
+          id={`${test.title}-header`}
+          expandIcon={view === 'run' && <ExpandMoreIcon />}
+        >
+          <ListItem className={styles.testCardList}>
+            {resultIcon}
+            {testText}
+            {messagesBadge}
+            {requestsBadge}
+            {testRunButton}
+          </ListItem>
+        </AccordionSummary>
+        <Divider />
+        <AccordionDetails
```

Comment:
The report view is extremely slow now that these are rendered in teh report view.  Plus they don't make sense as-is because only one of the tabs is shown at once which doesn't work for printing.  Can you keep this behavior the same as it was before and do not render these in the report?  

![Screen Shot 2022-02-25 at 3 57 50 PM](https://user-images.githubusercontent.com/412901/155802930-b3ea6029-c3d7-47f9-b2c5-785ff7c72881.png)

Reply:
Should be working now! Not sure if it's still too slow though

Reply:
Looks good.  Much better than before, and I think about equivalent to what is on main performance-wise (which as we know isn't great but we have ideas on how to improve that).